frontend/src/metabase/query_builder/components/view/ViewSidebar.jsx

```diff
@@ -1,41 +0,0 @@
-/* eslint-disable react/prop-types */
-import React from "react";
-import cx from "classnames";
-import { Motion, spring } from "react-motion";
-
-const SPRING_CONFIG = { stiffness: 200, damping: 26 };
-
-const ViewSideBar = ({ left, right, width = 355, isOpen, children }) => (
-  <Motion
-    defaultStyle={{ opacity: 0, width: 0 }}
-    style={
-      isOpen
-        ? { opacity: spring(1), width: spring(width, SPRING_CONFIG) }
```

Comment:
@gusaiani Please also note that there would be difference in the way how it animates:
- previously `spring`
- now `linear`